### ISSUE TITLE
Changing docked native tool settings widget state does not work

### ISSUE THREAD

**Issue by GintV** [iTwin/appui#372]
**Describe the bug**

Let's say I define toolSettings like so in frontstage config

```
toolSettings: {
  id: "NativeToolSettingsWidget",
  isToolSettings: true,
},
```
 
and then if I try to hide it like so
 
```
UiFramework.frontstages.activeFrontstageDef?.findWidgetDef("NativeToolSettingsWidget")?.
  setWidgetState(WidgetState.Hidden);
```
 
it does nothing. Tool settings widget is still there.
 
But, if I manually undock (make it floating) first and then try to hide with the code above, it dissapears.

**Expected behavior**
Docked tool settings should be treated as any other widget and react to setWidgetState.

**Desktop (please complete the applicable information):**

- OS: win 10
- Browser: chrome
- Version: 98.0.4758.141
- iTwin.js AppUI Version: 3.7.6
- iTwin.js Version: 3.7.6

**Comment by GintV** [comment]
I think it is a bug

**Comment by arnoldas-v** [comment]
Since appui 4 the floating tool settings widget does not even disappear anymore when changing state to `WidgetState.Hidden`. Also `UiFramework.frontstages.activeFrontstageDef?.findWidgetDef` does not find tool settings by id anymore as well.
I would expect tool settings widget to be treated as other widgets (found by findWidgetDef and react to state changes).

**Comment by GintV** [comment]
@GerardasB any news on this one? Is there any way this could be prioritized? This is really needed for our team

**Comment by GerardasB** [comment]
Not yet, I'll prioritize this for next week.

**Comment by raplemie** [comment]
Hi, a fix for this have been included and released in 4.3.0. Let us know if it addresses your issue as expected!
